src/vs/workbench/contrib/inlineChat/browser/inlineChatController.ts

```diff
@@ -271,6 +273,7 @@ export class InlineChatController implements IEditorContribution {
 
 		} catch (error) {
 			// this should not happen but when it does make sure to tear down the UI and everything
+			this._log('error during run', error);
```

Comment:
Kinda double logging because `onUnexpectedError` also writes to the log channel. Tho, w/o the prefix

Reply:
it's just when tracing is enabled, it's great to filter by prefix, which leads to missing non-prefixed errors; kept it for now, but happy to remove if you have strong opinion about this